src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -48,23 +48,25 @@ public class ReleaseService : IReleaseService
         private readonly IImportStatusService _importStatusService;
 	    private readonly IFootnoteService _footnoteService;
         private readonly IDataBlockService _dataBlockService;
+        private readonly IGuidGenerator _guidGenerator;
 
         // TODO PP-318 - ReleaseService needs breaking into smaller services as it feels like it is now doing too
         // much work and has too many dependencies
         public ReleaseService(
-            ContentDbContext context, 
-            IMapper mapper, 
+            ContentDbContext context,
+            IMapper mapper,
             IPublishingService publishingService,
-            IPersistenceHelper<ContentDbContext> persistenceHelper, 
-            IUserService userService, 
-            IReleaseRepository repository, 
+            IPersistenceHelper<ContentDbContext> persistenceHelper,
+            IUserService userService,
+            IReleaseRepository repository,
             ISubjectService subjectService,
-            ITableStorageService coreTableStorageService, 
-            IFileStorageService fileStorageService, 
+            ITableStorageService coreTableStorageService,
+            IFileStorageService fileStorageService,
             IImportStatusService importStatusService,
-	        IFootnoteService footnoteService, 
+            IFootnoteService footnoteService,
             StatisticsDbContext statisticsDbContext,
-            IDataBlockService dataBlockService)
+            IDataBlockService dataBlockService, 
+            IGuidGenerator guidGenerator)
```

Comment:
Not something we need to do for this ticket, but this class is definitely feeling like it's becoming harder to maintain with its continuously growing dependency list. Feels like we should act on the comment above at some point soon.

Reply:
There's already a ticket for this, EES-212. I will update the reference.